dev/ci-test.sh

```diff
@@ -53,9 +52,27 @@ $GITHUB_WORKSPACE/dev/ci-build.sh
 # mvn -Dtest=none -Dmaven.test.skip=false test
 
 # Individual test
-mvn --no-transfer-progress -Dtest=none -DwildcardSuites=org.apache.spark.ml.clustering.IntelKMeansSuite test
-mvn --no-transfer-progress -Dtest=none -DwildcardSuites=org.apache.spark.ml.feature.IntelPCASuite test
+# mvn --no-transfer-progress -Dtest=none -DwildcardSuites=org.apache.spark.ml.clustering.IntelKMeansSuite test
+# mvn --no-transfer-progress -Dtest=none -DwildcardSuites=org.apache.spark.ml.feature.IntelPCASuite test
 # mvn -Dtest=none -DwildcardSuites=org.apache.spark.ml.recommendation.IntelALSSuite test
 
+
+SupportedSparkVersions=("Spark-3.0.0" "Spark-3.0.1" "Spark-3.0.2" "Spark-3.1.1")
```

Comment:
I think use spark-3.0.0 will be better. Need to use Spark in the documentation, not for the code.